src/globus_sdk/authorizers/refresh_token.py

```diff
@@ -78,14 +77,14 @@ def __init__(
 
         super().__init__(access_token, expires_at, on_refresh)
 
-    def _get_token_response(self) -> globus_sdk.OAuthTokenResponse:
+    def _get_token_response(self) -> globus_sdk.RfreshTokenResponse:
```

Comment:
Why was this not getting caught by our linters?

```suggestion
    def _get_token_response(self) -> globus_sdk.RefreshTokenResponse:
```

Reply:
Found it. There's an issue with the lazy importer code that makes `mypy` ignore unknown attributes. I'll do a separate PR with a relevant test.